observability/opentelemetry/src/main/java/org/wildfly/extension/opentelemetry/OpenTelemetrySubsystemRegistrar.java

```diff
@@ -184,31 +248,26 @@ static void validateExporter(OperationContext context, String exporter) throws O
         }
     }
 
-    @Override
-    public Collection<AttributeDefinition> getAttributes() {
-        return Arrays.asList(ATTRIBUTES);
-    }
-
-    @Override
-    public void registerAttributes(ManagementResourceRegistration resourceRegistration) {
-        OperationStepHandler handler = new ValidateExporterWriteAttributeHandler();
-        for (AttributeDefinition attr : getAttributes()) {
-            if(!attr.getFlags().contains(AttributeAccess.Flag.RESTART_ALL_SERVICES)) {
-                throw ControllerLogger.ROOT_LOGGER.attributeWasNotMarkedAsReloadRequired(attr.getName(), resourceRegistration.getPathAddress());
-            }
-            resourceRegistration.registerReadWriteAttribute(attr, null, handler);
-        }
-    }
+//    public void registerAttributes(ManagementResourceRegistration resourceRegistration) {
+//        OperationStepHandler handler = new ValidateExporterWriteAttributeHandler();
+//        for (AttributeDefinition attr : getAttributes()) {
+//            if(!attr.getFlags().contains(AttributeAccess.Flag.RESTART_ALL_SERVICES)) {
+//                throw ControllerLogger.ROOT_LOGGER.attributeWasNotMarkedAsReloadRequired(attr.getName(), resourceRegistration.getPathAddress());
+//            }
+//            resourceRegistration.registerReadWriteAttribute(attr, null, handler);
+//        }
+//    }
 
-    static class WildFlyOpenTelemetryConfigSupplier implements Supplier<OpenTelemetryConfig>, Consumer<OpenTelemetryConfig> {
-        private OpenTelemetryConfig config;
+    static class WildFlyOpenTelemetryConfigSupplier implements Supplier<WildFlyOpenTelemetryConfig>,
```

Comment:
Where is this used?

Reply:
I don't think it is. If not, it will go away before this PR is final